src/main/java/ru/ayaz/User.java

```diff
@@ -4,6 +4,7 @@
 import java.util.Set;
 
 public class User {
+
     private String nickname;
```

Comment:
&#x1F34E; **Issue**
It is better to initialize your object in its constructor and not a setter, cause initializing through setters could lead to objects which are not fully initialized (either you forgot to initialize, or some other shit happened).